dwst/scripts/test/particles.js

```diff
@@ -212,12 +212,139 @@ describe('particles module', () => {
         '${foo)}',
         '${foo()',
         '\\a',
+        'a\\',
```

Comment:
We should probably remove this combined test as we now have more accurate tests for each case.